frontend/src/components/LandingPage/Features/Features.test.js

```diff
@@ -0,0 +1,11 @@
+import Features from './Features'
+import { hasBeenRendered } from "../../../global";
+import { shallow } from "enzyme";
+
+const wrapper = shallow(<Features />);
+
+describe("Test Features Component", () => {
+    it("should render features component", () => {
+        hasBeenRendered(wrapper);
+    });
```

Comment:
I recommend adding the test cases below for more precise testing.
```
    it("should have 4 features", () => {
        expect(wrapper.find(Col)).toHaveLength(4);
    });

    it("should have Guess Price of Stock", () => {
        const { feature, desc } = featuresArray[0];
        hasText(wrapper, CardHeader, 0, feature);
        hasText(wrapper, CardText, 0, desc)
    });

    it("should have Stock Information", () => {
        const { feature, desc } = featuresArray[1];
        hasText(wrapper, CardHeader, 1, feature);
        hasText(wrapper, CardText, 1, desc)
    });

    it("should have Stock News Information", () => {
        const { feature, desc } = featuresArray[2];
        hasText(wrapper, CardHeader, 2, feature);
        hasText(wrapper, CardText, 2, desc)
    });

    it("should have Weather", () => {
        const { feature, desc } = featuresArray[3];
        hasText(wrapper, CardHeader, 3, feature);
        hasText(wrapper, CardText, 3, desc)
    });
```

Reply:
Added the tests. Thank you for writing them!